### PR TITLE
more pointers

### PR DESCRIPTION
**Pull request by KaiTardor**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (ShehrozIrfan) [inline .vscode/settings.json]: Change: Remove this file, we don't need this.

**Review by ShehrozIrfan — CHANGES_REQUESTED** [review]
Good work, interesting to see the pointers in action.

Just need a minor change to remove the `.vscode/settings.json` file, because it's not needed.

File: .vscode/settings.json (hunk 1 above)
Comment (KaiTardor) [reply .vscode/settings.json]: Okey

**Review by ShehrozIrfan — APPROVED** [review]
LGTM

**Comment by ShehrozIrfan** [comment]
@KaiTardor 

Perfect 🚀

You can follow me and star this repo for further updates.
Have fun!

**Pull request merged**